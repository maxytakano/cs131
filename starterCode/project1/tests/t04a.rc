function: void _foo() {
    int i;
    bool b;
    float f;

    if (i) {
        i;
    }
    
    if (f) {
        f;
    }

    if (b) {
        i;
    }
}
