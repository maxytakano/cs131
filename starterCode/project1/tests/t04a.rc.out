Error, "/tmp/cs131_testrunner_5CTaux/file.rc", line 6: 
  bool required for conditional test, int found.
Error, "/tmp/cs131_testrunner_5CTaux/file.rc", line 10: 
  bool required for conditional test, float found.
Compile: failure.
